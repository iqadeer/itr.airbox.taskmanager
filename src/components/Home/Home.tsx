import React from 'react';
import { Container, Grid } from '@material-ui/core';
import Typography from '@material-ui/core/Typography';

const Home = () => {
  return (
    <Container>
      <Grid container justify='center' alignItems='center' spacing={2}>
        <Grid item xs={12} md={10}>
          <Typography variant='h4'>Welcome to Airbox task manager</Typography>
          <Typography variant='h6'>Choose the links above to manage apropriate tasks</Typography>
        </Grid>
      </Grid>
    </Container>
  );
};

export default Home;
