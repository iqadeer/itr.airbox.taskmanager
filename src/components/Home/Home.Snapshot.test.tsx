import React from 'react';
import renderer from 'react-test-renderer';
import Home from './Home';

it('should have correct text', () => {
  const home = renderer.create(<Home></Home>);
  expect(home).toMatchSnapshot();
});
