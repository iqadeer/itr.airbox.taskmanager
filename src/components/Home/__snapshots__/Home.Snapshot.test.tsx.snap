// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should have correct text 1`] = `
<div
  className="MuiContainer-root MuiContainer-maxWidthLg"
>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-10"
    >
      <h4
        className="MuiTypography-root MuiTypography-h4"
      >
        Welcome to Airbox task manager
      </h4>
      <h6
        className="MuiTypography-root MuiTypography-h6"
      >
        Choose the links above to manage apropriate tasks
      </h6>
    </div>
  </div>
</div>
`;
